src/main/resources/application.properties.example

```diff
@@ -5,4 +5,18 @@ spring.datasource.password=${RDS_PASSWORD}
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
 spring.logging.level.org.hibernate.SQL=debug
-spring.jpa.show-sql=true
\ No newline at end of file
+spring.jpa.show-sql=true
+
+
+#Google Calendar API Configuration
+google.client.client-id=${CLIENT_ID}
+google.client.client-secret=${CLIENT_SECRET}
+google.client.access-token-uri=https://www.googleapis.com/oauth2/v3/token
+google.client.user-authorization-uri=https://accounts.google.com/o/oauth2/auth?access_type=offline&prompt=consent
+google.client.client-authentication-scheme=query
+google.client.scope=profile,email,https://www.googleapis.com/auth/calendar
+google.resource.user-info-uri=https://www.googleapis.com/oauth2/v2/userinfo
+google.resource.prefer-token-info=true
+#google.client.redirectUri=http://localhost:9000/login/google
+
```

Comment:
If we store client credentials in a json file then we don't need this, please correct.